### PR TITLE
Add detailed description on README

### LINKED ISSUE #3: Update README.md
I feel there is no much information provided by README.md.
We might need to update it to let members easily understand this registration bot.

### PR DESCRIPTION
**Pull request by Darkborderman**
# Description
Add Update development and how to use on `README.md`.

# Fixes
fixes #3 .

### CODE REVIEW

**Review by jordanSu — APPROVED** [review]
LGTM! Thank you
Let me invite @josix , do you have any suggestions about what we can have for README?

File: README.md [hunk 1]
```diff
@@ -33,15 +33,40 @@ pipenv install --dev
...
+
+### Tokens
+Tokens should be stored under "tokens" folder.
+
+Because ticket has different types, there are three different CSV files
+* `corporate.csv` for corporate tickets
+* `individual.csv` for individual tickets
+* `reserved.csv` for reserved tickets
+
+to represent each ticket type.
```
Comment (josix) [inline README.md]: NIT. I guess the order of the statements could be exchanged as follows:
```suggestion
to represent each ticket type:
* `corporate.csv` for corporate tickets
* `individual.csv` for individual tickets
* `reserved.csv` for reserved tickets
```

**Review by josix — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (Darkborderman) [reply README.md]: Resolved.

**Review by josix — APPROVED** [review]
LGTM, thanks @Darkborderman !

**Pull request merged**